Alignment/OfflineValidation/test/DiMuonVertexValidation_cfg.py

```diff
@@ -56,7 +57,7 @@ def best_match(rcd):
                   "seed number")
 
 options.register ('myfile',
-                  '/store/relval/CMSSW_10_6_1/RelValZMM_13/GEN-SIM-RECO/PU25ns_106X_mc2017_realistic_v6_HS-v1/10000/44690279-DDF3-0D43-B92D-F5CB57EF7E6A.root', # default value
+                  filesDefaultMC_Realistic2022_string, # default value
```

Comment:
why are you changing Z->mm for MinBias here?

Reply:
Yes, but it is for testing purpose only, previous samples were not there anymore... didn't want to break PR process with waiting for rucio rule to go through.

Reply:
No the samples are cached. It doesn't make sense to run something called `DiMuon...` on MinimumBias . Please don't change that.

Reply:
ok I changed for `/store/relval/CMSSW_12_5_0_pre5/RelValZMM_14/GEN-SIM-RECO/125X_mcRun3_2022_realistic_v3-v1/10000/068634c7-e8e2-4e46-a68e-97ebefed4868.root`